tests/system/test_instance_admin.py

```diff
@@ -614,3 +652,69 @@ def test_cluster_update(
     admin_cluster.serve_nodes = serve_nodes
     operation = admin_cluster.update()
     operation.result(timeout=60)  # Ensure the operation completes.
+
+
+def test_cluster_update_w_autoscaling(
+    admin_instance_populated,
+    admin_cluster_id,
+    admin_cluster_with_autoscaling,
+    min_serve_nodes,
+    max_serve_nodes,
+    cpu_utilization_percent,
+    skip_on_emulator,
+):
+    new_min_serve_nodes = min_serve_nodes + 1
+    new_max_serve_nodes = max_serve_nodes + 1
+    new_cpu_utilization_percent = cpu_utilization_percent + 10
+    admin_cluster_with_autoscaling.min_serve_nodes = new_min_serve_nodes
+    admin_cluster_with_autoscaling.max_serve_nodes = new_max_serve_nodes
+    admin_cluster_with_autoscaling.cpu_utilization_percent = new_cpu_utilization_percent
+
+    operation = admin_cluster_with_autoscaling.update()
+    operation.result(timeout=60)  # Ensure the operation completes.
+
+    # Create a new cluster instance and reload it.
+    alt_cluster = admin_instance_populated.cluster(admin_cluster_id)
+    alt_cluster.reload()
+    assert alt_cluster.min_serve_nodes == new_min_serve_nodes
+    assert alt_cluster.max_serve_nodes == new_max_serve_nodes
+    assert alt_cluster.cpu_utilization_percent == new_cpu_utilization_percent
```

Comment:
let's add one more test case in which only one field is updated.

Reply:
I've added such test below (line 691)